### PR TITLE
#1231 HTML Lists Updated  README.md

### PR DESCRIPTION
**Pull request by Elisha-s**
<hr>

## Description 📜

I have documented 3 types of HTML Lists along with the code and the output pictures. 
Fixes   <https://github.com/girlscript/winter-of-contributing/issues/1231> 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
 

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/HTML_LISTS/README.md [hunk 1]
```diff
@@ -1,69 +1,109 @@
...
 </ol>
-```
 
-On a web page this type of list looks something like this :
+The ordered list marker is numeric by default,however we can always use the ```type``` attribute to define the type of marker that we want in our ordered list. The markers can be numbers, upper and lower case letters and upper and lower case roman numbers.They follow the syntax:
+``` <ol type="a"></ol> ``` 
+
+---
 
-![](./assets/ordered_list.png)
```
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/HTML_LISTS/README.md:23]: Please add a screenshot of the output as well for each and every type of list.

File: Web_Development/FrontEnd/HTML_LISTS/README.md [hunk 2]
```diff
@@ -1,69 +1,109 @@
...
+    <dt>Prateek</dt>
+    <dd>Rank 1</dd>
+   <dt>Amit</dt>
+    <dd>Rank 2</dd>
+   <dt>Sara</dt>
+    <dd>Rank 3</dd>
+</dl>
+
+Using HTML lists not only organises our data, it also helps us style our website better because all the list items are independent and they can be styled differently using different CSS styles. We can also move the data around easily at any point of time by changing the order of our code. This can be a simple cut and paste, so they are very 
+easy to maintain as well.
```
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/HTML_LISTS/README.md:106]: Keep this in same line - 105.

**Review by Aayushi-Mittal — APPROVED** [review]
Great work @Elisha-s. 👏
Please do the requested changes. Rest looks good to me!

**Comment by Elisha-s** [comment]
> Great work @Elisha-s. 👏 Please do the requested changes. Rest looks good to me!

I did all the changes required. Please check my pull request sir @praveenscience  . I am from Batch 17 of Frontend Web Development using HTML,CSS and JS.

**Comment by sifa123** [comment]
> ## Description 📜
> I have documented 3 types of HTML Lists along with the code and the output pictures. Fixes # <1231>
> 
> ## Type of change 📝
> * [x]  Doc#umentation (Content Creation in the form of codes or tutorials)
> 
> ## Domain of Contribution 📊
> * [x]  Frontend Dev #HTML/CSS/JS
> 
> ## Checklist ✅
> * [x]  I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
> * [x]  I have performed a self-review of my own code or work.
> * [x]  My changes generates no new warnings.
> * [x]  I'm GWOC'21 contributor
> 
> ## Screenshots / Gif (Optional) 📸

In Description box please fix the issue link

**Review by sifa123 — COMMENTED** [review]
Excellent @Elisha-s

**Comment by Elisha-s** [comment]
> > ## Description 📜
> > I have documented 3 types of HTML Lists along with the code and the output pictures. Fixes # <1231>
> > ## Type of change 📝
> > 
> > * [x]   Doc#umentation (Content Creation in the form of codes or tutorials)
> > 
> > ## Domain of Contribution 📊
> > 
> > * [x]   Frontend Dev #HTML/CSS/JS
> > 
> > ## Checklist ✅
> > 
> > * [x]   I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
> > * [x]   I have performed a self-review of my own code or work.
> > * [x]   My changes generates no new warnings.
> > * [x]   I'm GWOC'21 contributor
> > 
> > ## Screenshots / Gif (Optional) 📸
> 
> In Description box please fix the issue link

Done.

**Comment by Elisha-s** [comment]
> Excellent @Elisha-s

Thank you. How am I supposed to proceed further?

**Pull request merged**